src/Nethermind/Nethermind.DataMarketplace.Consumers.Test/Infrastructure/NdmRpcConsumerModuleTests.cs

```diff
@@ -546,7 +546,7 @@ private static DepositDetails GetDepositDetails()
         private static DepositReportItem GetDepositReportItem()
             => new DepositReportItem(Keccak.Zero, TestItem.KeccakA, "test", TestItem.AddressA,
                 "test", 1, 1, TestItem.AddressB, 1, 1, false, TestItem.KeccakA,
-                1, 1, 1, true, false, TestItem.KeccakB, 1, new[]
+                1, 1, 1, true, false, TestItem.KeccakB, false, 1, new[]
```

Comment:
create a test that would fail without a fix and does not fail with the fix, is that reasonable?